lib/src/disposable_manager/disposable_manager_vm.dart

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
I like the suggestion of `_common` rather than `_vm` for the disposable file. So maybe we should do the same thing here. Or maybe group them together in a `common` directory?